src/sandstorm/spk.c++

```diff
@@ -1093,7 +1093,12 @@ private:
         "  argv = [", argv, "],\n"
         "  environ = [\n"
         "    # Note that this defines the *entire* environment seen by your app.\n"
-        "    (key = \"PATH\", value = \"/usr/local/bin:/usr/bin:/bin\")\n"
+        "    (key = \"PATH\", value = \"/usr/local/bin:/usr/bin:/bin\"),\n"
+        "    (key = \"SANDSTORM\", value = \"1\"),\n"
+        "    # Apps that are adapted for Sandstorm may benefit from enabling additional\n"
```

Comment:
I find this a little hard to read, so I propose the following:

```
Export SANDSTORM=1 into the environment, so that apps running within Sandstorm
can detect if $SANDSTORM == "1" at runtime, switching UI and/or backend to use
the app's Sandstorm-specific integration code.
```